test/integration/transactionsAPITests.coffee

```diff
@@ -272,14 +273,32 @@ describe "API Integration Tests", ->
                   done()
 
       it "should call getTransactions with filter paramaters ", (done) ->
```

Comment:
It would be good if there was a bit more comprehensive test coverage of the different filter param options.